File: swift/medium_com_rubik.swift
// https://www.codingame.com/training/medium/rubik%C2%AE

import Glibc
import Foundation

public struct StderrOutputStream: TextOutputStream {
    public mutating func write(_ string: String) { fputs(string, stderr) }
}
public var errStream = StderrOutputStream()

let N = Int(readLine()!)!
var ans = 0;
if (N > 1) {
    ans = 6 * N * (N - 2) + 8;
} else {
    ans = 1;
}
print(ans);
// Write an action using print("message...")
// To debug: print("Debug messages...", to: &errStream)

File: lua/medium_com_rubik.lua
-- https://www.codingame.com/training/medium/rubik%C2%AE

N = tonumber(io.read())
if (N > 1) then
    ans = 6 * N * (N - 2) + 8;
else
    ans = 1;
end
print(ans)
-- Write an action using print()
-- To debug: io.stderr:write("Debug message\n")

-- os.execute("python3 -c\"n=int(input());print(6*n*(n-2)+8 if n>1 else 1)\"")
